repository/rest-proxy/operator/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: rest-proxy
+spec:
+  replicas: 1
```

Comment:
let's parametrize that

Reply:
Done. Added parameters to configure this.